### DISCUSSION TITLE
1.20.4?

### DISCUSSION THREAD

**Discussion by ItzMahmood** [Sinytra/ForgifiedFabricAPI#122] [Q&A] [answered]
please, can you release the 1.20.4 version of your mods? specifically these mods Forgified Fabric API & Sinytra Connector.

**Comment by MSandro** [comment, marked as answer]
A 1.20.4 NeoForge port is in the works, I hope that it will be ready soon and not only appear after the release of MC 1.21.

**Comment by Vitokhv** [comment]
Yes, need 1.20.4

**Reply by coco875** [reply]
finally it's get skipped
